visualization/app/codeCharta/util/fileValidator.ts

```diff
@@ -1,15 +1,62 @@
 import { CodeMapNode } from "../codeCharta.model"
+import * as codeCharta from "../../../package.json"
+import { DialogService } from "../ui/dialog/dialog.service"
 
 export class FileValidator {
+	public static validate(file: { nodes: CodeMapNode[] }, dialogService?: DialogService): string[] {
+		let minorApiWrongMessage = ""
+
+		if (!file) {
+			return ['<i class="fa fa-exclamation-circle"></i>' + " file is empty or invalid"]
+		}
+
+		let Validator = require("jsonschema").Validator
```

Comment:
how about importing the `Validator` using new syntac. Does that work?
`import { Validator } from "jsonschema"`

Reply:
i found no way to do it like that: but i can shorten it to 
`let validationResult = require("jsonschema").validate(file, `require("./schema.json"))`

Reply:
Okey sad, then let’s keep it, but can you put the required statement to the imports?